### DISCUSSION TITLE
'NoneType' object has no attribute 'axes'

### DISCUSSION THREAD

**Discussion by vishals2911** [pycaret/pycaret#1310] [Q&A] [answered]
I have trained model on colab and now trying to use the model on my local but am getting below error.
Am using 2.3.1 version on colab as well as on local.

![image](https://user-images.githubusercontent.com/68744243/120058575-82978800-c069-11eb-94cf-6b0b11cd84f0.png)

![image](https://user-images.githubusercontent.com/68744243/120058564-685daa00-c069-11eb-89c8-576941d04e88.png)

**Comment by Yard1** [comment, marked as answer]
Is it possible for you to share the model and dataset?

**Reply by vishals2911** [reply]
Hi, i have uploaded the dataset and pickle fine at below location.

https://drive.google.com/drive/folders/1hiXxCM9648DXxolMSO15iFX6eWUtM_Jl?usp=sharing

**Reply by Yard1** [reply]
I couldn't replicate the issue locally. Are you using the same version of PyCaret on both Colab and local?

![image](https://user-images.githubusercontent.com/10364161/120084840-ca52f980-c0d3-11eb-8e34-b9d1a8023871.png)

**Reply by vishals2911** [reply]
> Thanks, able to solve this...it was system specific issue.
